llarp/config/config.hpp

```diff
@@ -105,7 +105,7 @@ namespace llarp
     size_t m_minConnectedRouters = 2;
 
     /// hard upperbound limit on the number of router to router connections
-    size_t m_maxConnectedRouters = 2000;
+    size_t m_maxConnectedRouters = 5;
```

Comment:
Was this intentional?  5 seems really low.

Reply:
remember this applies to clients too